pkg/kubelet/cm/cpuset/cpuset.go

```diff
@@ -57,7 +57,7 @@ func (b Builder) Add(elems ...int) {
 
 // Result returns the result CPUSet containing all elements that were
 // previously added to this builder. Subsequent calls to Add have no effect.
-func (b Builder) Result() CPUSet {
+func (b *Builder) Result() CPUSet {
```

Comment:
@tiloso static check is fixed by this PR
https://github.com/kubernetes/kubernetes/pull/103723

Reply:
I am not sure if this is the right thing to do here. I say this because when you look at the implementation of a consumer does not mix and match pointer types. Changing this would mean Builder.Add acts on an object, Builder.Result acts on a receiver to pointer.

https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/cm/cpumanager/topology/topology.go#L83-L89